publication-commons/src/main/java/no/unit/nva/publication/service/impl/ReadResourceService.java

```diff
@@ -93,14 +93,13 @@ private String constructPrimaryPartitionKey(UserInstance userInstance) {
             .collect(Collectors.toList());
     }
 
-    public Publication getPublicationByIdentifier(SortableIdentifier identifier) throws NotFoundException {
+    public Resource getResourceByIdentifier(SortableIdentifier identifier) throws NotFoundException {
```

Comment:
change the method to return Resource in order to have access to the row version. The method is used by the getPublicationByIdentifier that maintains the expected behavior.